FMS/Pages/Facilities/Details.cshtml

```diff
@@ -31,87 +31,153 @@
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.Cabinets, "Cabinets", new {Model.FacilityDetail.IsRetained})
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.FacilityStatus)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.FacilityStatus.Status, "StringOrNone")
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.FacilityType)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.FacilityType.Name, "FacilityTypeIcon")
-            @Html.DisplayFor(model => model.FacilityDetail.FacilityType.Description, "StringOrNone")
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.BudgetCode)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.BudgetCode.Name, "StringOrNone")
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.OrganizationalUnit)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.OrganizationalUnit.Name, "StringOrNone")
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.ComplianceOfficer)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.ComplianceOfficer.Name, "StringOrNotAssigned")
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.Location)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.Location, "StringOrNone")
-        </dd>
-        <dt class="col-sm-3">Address</dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.Address)<br />
-            @Html.DisplayFor(model => model.FacilityDetail.City),
-            @Html.DisplayFor(model => model.FacilityDetail.State)
-            @Html.DisplayFor(model => model.FacilityDetail.PostalCode)
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.County)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.County.Name, "StringOrNone")
-        </dd>
-        <dt class="col-sm-3">
-            @Html.DisplayNameFor(model => model.FacilityDetail.Latitude)/@Html.DisplayNameFor(model => model.FacilityDetail.Longitude)
-        </dt>
-        <dd class="col-sm-9">
-            @Html.DisplayFor(model => model.FacilityDetail.Latitude),
-            @Html.DisplayFor(model => model.FacilityDetail.Longitude)
-        </dd>
-    </dl>
+<div class="container-fluid">
```

Comment:
See if you can make this two-column layout collapse to a single column at smaller screen sizes.

Reply:
Corrected this. I had it working then made changes to the containers and forgot to retest the sizing. 
Thanks for picking that up.